views/signedup.tmpl

```diff
@@ -0,0 +1,100 @@
+      <div class="col">
+        <h1 class="display-3">On Air, Online, On Demand</h1>
+        <h2 class="display-4">Broadcasting since 1968.</h2>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container-fluid container-padded bg-primary">
+	<div class="container container-padded">
+{{if .Feedback}}
+    <h1 class="text-center">Unsuccessful Sign up:</h1>
+    <div class="row">
+      <div class="col-md-3"></div>
+      <div class="col-md-6">
+        <ul>
+          {{range .Feedback}}
+            <li>{{.}}</li>
+          {{end}}
+        </ul>
+        <br>
+        <p class="text-center"><a href="/getinvolved#getInvolvedForm">&lt; Back to the form</a></p>
+      </div>
+      <div class="col-md-3"></div>
+    </div>
+  </div>
+</div>
+{{else}}
+    <h1 class="text-center">Success!</h1>
+    <h3 class="text-center">We've registered you on our computers, and you'll soon get an email telling you all about what happens next!</h3>
+    <p class="text-center">For now, here's some other useful things to do:</p>
+    <div class="row">
+      <div class="col-md-1"></div>
+      <div class="col-md-10">
+      </div>
+      <div class="col-md-1"></div>
+    </div>
+  </div>
+</div>
+<div class="container-fluid container-padded bg-off-white">
+  <div class="container container-padded">
+    <div class="row"> <br> </div>
+    <div class="row">
+
+      <div class="col-md-3 bg-primary rounded">
+        <br>
+        <img src="/images/slack-logo.svg" class="img-fluid" alt="Slack Logo">
+        <p></p>
+        <p class="text-center">Find the URY slack team at <a href="http://ury.slack.com">ury.slack.com</a> and join all the channels you're interested in.</p>
+      </div>
+
+      <div class="col-md-1"><br></div>
+
+      <div class="col-md-4 bg-primary">
+        <br>
+        <h1 class="text-center">Come see us!</h1>
+        <br>
+        <p class="text-center">We have a calendar of <a href="https://goo.gl/5APK7Q">all our meetings</a>, to which new members are always welcome, and you can drop by the station almost any time and there'll be someone to talk to.</p>
+        <br>
+        <p class="text-center">All of our team meetings are held in Vanbrugh Arms (V&nbsp;Bar), which is right next to the radio station</p>
+        <br>
+        <a class="text-center" href="/contact#map"><h3>Map of where to find us</h3></a>
+      </div>
+
+      <div class="col-md-1"><br></div>
+
+      <div class="col-md-3 bg-primary rounded">
+        <br>
+        <img src="https://ury.org.uk/myradio/img/myradio-logo.svg" class="img-fluid rounded mx-auto d-block" alt="MyRadio Logo" style="margin-top: 30%; margin-bottom: 30%">
+        <p></p>
+        <p class="text-center">Once you get your confirmation email, you'll be able to log in to <a href="https://ury.org.uk/myradio/">MyRadio</a>. This is where you'll be able to book training, request shows and do other magical things.</p>
+      </div>
+
+    </div>
+  </div>
+</div>
+{{end}}
+
+{{end}}
+{{end}}
```

Comment:
add an empty line at the end of this file

Reply:
If you're using VScode, try this in your user settings :) https://gist.github.com/mstratford/a7515d37a01ee83eb6d84530942ab103

Reply:
Check the read me, I have now added details about the .editorconfig file, this should help